islandora.module

```diff
@@ -58,7 +58,11 @@ define('ISLANDORA_DATASTREAM_INGESTED_HOOK', 'islandora_datastream_ingested');
 define('ISLANDORA_DATASTREAM_MODIFIED_HOOK', 'islandora_datastream_modified');
 define('ISLANDORA_DATASTREAM_PURGED_HOOK', 'islandora_datastream_purged');
 define('ISLANDORA_INGEST_STEP_HOOK', 'islandora_ingest_steps');
-define('ISLANDORA_DERVIATIVE_CREATION_HOOK', 'islandora_derivative');
+define('ISLANDORA_DERIVATIVE_CREATION_HOOK', 'islandora_derivative');
+// ISLANDORA_DERVIATIVE_CREATION_HOOK is deprecated. Should be removed
+// as of 7.x-1.5.
+define('ISLANDORA_DERVIATIVE_CREATION_HOOK', 'ISLANDORA_DERIVATIVE_CREATION_HOOK');
```

Comment:
Should be the new constant itself as the second parameter (no quotation marks)... This has a string containing the _name_ of the new constant.